src/components/responsive.css

```diff
@@ -0,0 +1,144 @@
+/* responsive rules - starts here */
```

Comment:
Good idea to keep all the responsiveness-css in one file. If you want to use styled components you can put the mediaqueries for each of those in the same place. So you would have the basic styling at the top and then directly below inside the backticks, you can have the media queries for that component.